src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyApprovalServiceTests.cs

```diff
@@ -1026,7 +1188,128 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
                 Assert.Null(updatedMethodology.Published);
                 Assert.Equal(Draft, updatedMethodology.Status);
                 Assert.Equal(Immediately, updatedMethodology.PublishingStrategy);
-                Assert.Null(updatedMethodology.InternalReleaseNote);
+                Assert.Equal("A release note", updatedMethodology.InternalReleaseNote);
+                Assert.True(updatedMethodology.Updated.HasValue);
+                Assert.InRange(DateTime.UtcNow.Subtract(updatedMethodology.Updated!.Value).Milliseconds, 0, 1500);
```

Comment:
Do you not also wanna check to see that there are no MethodologyStatuses in the db at this point?

```
 Assert.Empty(await context
                    .MethodologyStatus
                    .WhereAsync(ms => ms.MethodologyVersionId == methodologyVersion.Id));
```

or similar?